SECURITY.md

```diff
@@ -11,12 +11,14 @@ We ask anyone who discovers a security concern associated with any of the
 rust-vmm components to send an encrypted email to the rust-vmm maintainers,
 using the public PGP keys located below.
 
-|Maintainer	|Email	|PGP Key	|
-|---	|---	|---	|
-|Andreea Florescu |fandree@amazon.com |https://github.com/andreeaflorescu.gpg |
-|Laura Loghin |lauralg@amazon.com	|https://github.com/lauralt.gpg	|
-|Rob Bradford |robert.bradford@intel.com |https://github.com/rbradford.gpg |
-|Sebastien Boeuf |sebastien.boeuf@intel.com |https://github.com/sboeuf.gpg |
+|Maintainer|Email|PGP Key|
+|---|---|---|
+|Patrick Roy|roypat@amazon.co.uk|https://github.com/roypat.gpg|
+|Jonathan Woollett-Light|jcawl@amazon.co.uk|https://github.com/JonathanWoollett-Light.gpg|
```

Comment:
Please use alphabetical order.

Reply:
I think we should add a check to explicitly enforce this, or not require it. Personally I don't see any clear benefit from it.

Reply:
It was alphabetical order before, and we've been using this across the project. I believe it's pretty standard to use alphabetical order unless there is a hierarchy or a particular order in which you need to contact people. I don't think we should have a check for something as simple as this. For example, the maintainers list in Firecracker is also alphabetical.

Reply:
Fixed